### PR TITLE
Add how to use plugin with nuxt to readme

### LINKED ISSUE #15: Ensure the plugin works when imported

### PR DESCRIPTION
**Pull request by axlwaii**
closes #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,10 @@ Vue.use(VueSafeHTML, {
 });
 ```
 
+### Nuxt
+
+`vue-safe-html` is written as Vue plugin so you can easily use it inside nuxt by following [the nuxt documentation](https://nuxtjs.org/docs/2.x/directory-structure/plugins#vue-plugins).
```
Comment (jakubfiala) [inline README.md:90]:
```suggestion
`vue-safe-html` is written as a Vue plugin so you can easily use it inside nuxt by following [the nuxt documentation](https://nuxtjs.org/docs/2.x/directory-structure/plugins#vue-plugins).
```

**Review by jakubfiala — COMMENTED** [review]
just a tiny style fix

File: README.md (hunk 1 above)
Comment (LostCrew) [reply README.md:90]: + I'd make `Nuxt` uppercase everywhere

Comment (axlwaii) [reply README.md]: Changed it :v:

File: README.md [hunk 2]
```diff
@@ -85,6 +85,10 @@ Vue.use(VueSafeHTML, {
 });
 ```
 
+### Nuxt
+
+`vue-safe-html` is written as a Vue plugin so you can easily use it inside nuxt by following [the Nuxt documentation](https://nuxtjs.org/docs/2.x/directory-structure/plugins#vue-plugins).
```
Comment (LostCrew) [inline README.md:90]:
```suggestion
`vue-safe-html` is written as a Vue plugin so you can easily use it inside Nuxt by following [the Nuxt documentation](https://nuxtjs.org/docs/2.x/directory-structure/plugins#vue-plugins).
```

**Review by jakubfiala — APPROVED** [review]
🔥

**Pull request merged**